post-install configuration (#62957)
openshift/openshift-docs: modules/virt-node-placement-rule-examples.adoc

                values:
                - 8 <3>
----
<1> Infrastructure resources are placed on nodes labeled `example.io/example-infra-key = example-value`.
<2> workloads are placed on nodes labeled `example.io/example-workloads-key = example-workloads-value`.
<3> Nodes that have more than eight CPUs are preferred for workloads, but if they are not available, pods are still scheduled.

.Example `HyperConverged` object with `tolerations` rule
[source,yaml,subs="attributes+"]
----
apiVersion: hco.kubevirt.io/v1beta1
kind: HyperConverged
metadata:
  name: kubevirt-hyperconverged
  namespace: {CNVNamespace}
spec:
  workloads:
    nodePlacement:
      tolerations: <1>
      - key: "key"
        operator: "Equal"
        value: "virtualization"
        effect: "NoSchedule"
----
<1> Nodes reserved for {VirtProductName} components are labeled with the `key = virtualization:NoSchedule` taint. Only pods with matching tolerations are scheduled on these nodes.

[id="hostpathprovisioner-object-node-placement-rules_{context}"]
== HostPathProvisioner object node placement rule example

You can edit the `HostPathProvisioner` object directly or by using the web console.

[WARNING]
====

> You must schedule the hostpath provisioner and the {VirtProductName} components on the same nodes. Otherwise, virtualization pods that use the hostpath provisioner cannot run.

Reviewer: Does this actually throw an error that has a corresponding runbook, or the VMs just never gets scheduled? If there is, please link it here.

Author: This warning was added for CNV 2.6. I will check this with Simone.

> You must schedule the hostpath provisioner and the {VirtProductName} components on the same nodes. Otherwise, virtualization pods that use the hostpath provisioner cannot run. You will not be able to run virtual machines.

Reviewer 2: ```suggestion
You must schedule the `HostPathProvisioner` and the {VirtProductName} components on the same nodes. Otherwise, virtualization pods that use the `HostPathProvisioner` cannot run. It is not possible to run virtual machines.
```

Reviewer 2: I think `HostPathProvisioner` needs to be in backticks. And also, switching the language here so it isn't future possible.
